src/Reducer/Reducing.js

```diff
@@ -0,0 +1,24 @@
+import { combineReducers } from 'redux'
+
+const taskReducer = (state = [], action) => {
+    switch (action.type) {
+        case 'ADD_TODO':
+            console.log(action.text)
```

Comment:
consoles should not be present when giving for final review. Take care of that further.